.serena/project.yml

```
@@ -1,90 +1,141 @@
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)